### DISCUSSION TITLE
Lists should appear publicly in Repositories Tab

### DISCUSSION THREAD

**Discussion by migueldinisdev** [community/community#9067] [Other Feature Feedback, Questions, & Ideas]
Hello!

I discovered this new feature which I really like because I am able to organize my profile's repos.

One thing I noticed is that whenever I add a repository to a list, it is automatically starred. Well, I would like to be able to make a list without having to star every single one of them. 

For example, I might want to group hobby projects in list A, academic projects in list B, research projects in list C and so on, and being able to star only a few of them. I don't think these two features correlate at all, actually :)

Moreover, lists are only visible to me (and a bit hidden to the public). I would like them to be visible in ["repositories" tab ](https://github.com/MigDinny?tab=repositories)

So, something like this:

![image](https://user-images.githubusercontent.com/6632800/146656705-3f7be872-db77-44dc-9015-98e049d57019.png)

Would also be present here:

![image](https://user-images.githubusercontent.com/6632800/146656710-5ff78d21-77e8-42cd-9817-8cc039775b86.png)

**Comment by brianlovin** [comment]
Thanks for the note and suggestions! Could you share a bit more about a case where you'd want to add a repository to a list but _not_ star it? Right now this functionality is coupled, but we've seen several requests to decouple it so I just want to make sure I understand more about the use case here. Thanks!

**Reply by willow-carroll-polzin** [reply]
I like to use the lists to sort various repo's and keep track of them, however I like to reserve the Star for repo's that I want to quickly access or am currently working with. Decoupling the star from the list would be great imo!

**Reply by BarakBinyamin** [reply]
I'd find this feature really helpful too, basically like youtube playlists. Right now a lot of people make a new repo just to consolidate a list of all the helpful projects out there

It could also be a new tab
<img width="717" alt="image" src="https://github.com/community/community/assets/50957106/dd3458f9-05ec-4a23-b6ab-05c2c19b7f85">

**Reply by Shanukuttoth** [reply]
Thanks for your support &helping. Asap more support from your side, may be am done. Knowledge is power 👍

**Comment by SantiagoRR2004** [comment]
I also agree with the idea of separating lists from stars. This topic also appears in https://github.com/orgs/community/discussions/8842 and https://github.com/orgs/community/discussions/125708.
